zerver/models.py

```diff
@@ -3123,6 +3123,7 @@ def sent_by_human(self) -> bool:
                 "website",
                 "ios",
                 "android",
+                "test suite",
```

Comment:
Thinking about this a bit more, I think we should add this only if `settings.TEST_SUITE`; going to try doing that change for this commit.